README.md

```diff
@@ -67,13 +67,13 @@ The PGP key is automatically generated and stored as a Kubernetes secret if it d
 ## Enforcers
 Enforcers are defined as [validating admission webhook](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/) that ensures the resource defined as an `image` in the `Pod` has been properly attested.
 
-Enforcers are configured to ensure all attesters referenced in the namespace for the pod have successfully created attestations.  The namespace must include a label for enforcement to be activated:
```

Comment:
this needs to remain as a label, not an annotation for the webhook to work...the webhook only fires for pods in namespaces with that label:

https://github.com/liatrio/rode/blob/master/helm-chart/rode/templates/secrets.yaml#L54-L57

Reply:
Is this not referencing a separate label? 
"rode.liatr.io/enforce-attesters" vs "liatr.io/rode"
My vauge understanding was that the first would determine which attesters the enforcer will take into account. Whereas the second would account for which namespaces the enforcer would be active in at all.

Perhaps I'm missing the required background here.

Reply:
Yep, you're right....there was uncommitted changes in my local workspace, my bad.  I changed `values.yaml` to use that first value.

Reply:
Great, believe I understand now.

No long term concerns about implicitly limiting the number of active attesters due to the character limit?

Reply:
Fair question.  I suspect that most namespaces will either require 1 specific attester, or just use `*` to require ALL attesters.  I just checked the spec and looks like `,` isn't even a valid character for a label value, so using the value as a comma separated list isn't gonna work :(

Open to suggestions here on how to proceed.

Reply:
I generally agree, because attesters can contain arbitrary numbers of policies (I think). I can't really think of any reason that you would _need_ more than one attester per namespace. However there may be some benefit due for logical distinctions of policy groups, along the lines of how attaching multiple roles to an account may be useful even though it wouldn't be necessary.

The only alternative that comes to mind immediately is a setup similar to how I initially understood, a label to enable enforcement on a particular namespace, and an annotation to set which attesters to enforce. A slight alternative would be to use a label to enable, and an annotation to set which attesters to **exclude** from enforcement. This has some design similars to other projects, which I find appealing. And it also minimizes configuration for users, assuming the most standard use case to enforcing all attesters, someone in that use case would only need to set a single label per namespace, matching the current design.

Reply:
Alternatively, we could attach a label to attesters, marking them as disabled or non-enforcable. If they're namespaced it should have roughly the same affect as marking the namespace

Reply:
Let's keep it simple for now:
* use the label for both webhook AND attester name
* support `*` for attester
* dont worry about `split` since `,` is not a valid character.